File: app/src/main/java/de/schooltec/datapass/datasupplier/CongstarDataSupplier.java
package de.schooltec.datapass.datasupplier;

/**
 * Class providing all necessary information of the German Congstar 'datapass' homepage.
 *
 * @author Martin Hellwig
 */
public class CongstarDataSupplier extends TelekomGermanyDataSupplier
{
    @Override
    public boolean isTelekomProvider() {
        return false;
    }
}

File: app/src/main/java/de/schooltec/datapass/datasupplier/CongstarDataSupplier.kt
package de.schooltec.datapass.datasupplier

/**
 * Class providing all necessary information of the German Congstar 'datapass' homepage.
 *
 * @author Martin Hellwig
 */
internal class CongstarDataSupplier : TelekomGermanyDataSupplier() {
    override val isTelekomProvider = false
}
